.travis.yml

```diff
@@ -3,8 +3,5 @@ dist: trusty
 language: bash
 env:
   - DOKKU_VERSION=master DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
-  - DOKKU_VERSION=v0.7.0 DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
-  - DOKKU_VERSION=v0.6.0 DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
-  - DOKKU_VERSION=v0.5.0 DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
-  - DOKKU_VERSION=v0.4.0 DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
+  - DOKKU_VERSION=v0.18.5 DOKKU_SYSTEM_GROUP=travis DOKKU_SYSTEM_USER=travis
```

Comment:
@josegonzalez suggested this because:
  - I was getting a lot of intermittent errors in all the different versions
  - Since we couple DDS and `dokku-acl` together in DDS, it would make sense to just test the current `v0.18.5` and the latest.